docker/README.md

```diff
@@ -0,0 +1,86 @@
+# Description
+This Docker Compose development environment includes
+
+* PHP 5.6.*
+* MySQL 5.6
+* Nginx 1.6.2
```

Comment:
I think you should create a `docker-compose.yml.dist`. 

When someone want to use docker with Sylius, he copy the dist to create the `docker-compose.yml`. Don't forget to add it in the gitignore.

Soon, docker-compose will use file "docker-compose.override.yml" to override configuration (https://github.com/docker/compose/blob/master/compose/config/config.py#L79).